src/Education/XamlGenerator/XamlGeneratorHelper.cs

```diff
@@ -144,10 +145,18 @@ private void WriteCleanCodeHeader(IRuleInfo ruleInfo)
                 writer.WriteStartElement("WrapPanel");
                 WriteCleanCodeHeader_CleanCodeAttribute(ruleInfo);
                 WriteCleanCodeHeader_SoftwareQualities(ruleInfo);
+                WriteCleanCodeHeader_Url();
                 writer.WriteEndElement();
                 writer.WriteEndElement();
             }
 
+            private void WriteCleanCodeHeader_Url()
+            {
+                writer.WriteStartElement("TextBlock");
```

Comment:
Does it break the link when the line overflows? Maybe add style to fix that if it's broken?